scripts/jenkins-yoctobuild-build.sh

```diff
@@ -1090,8 +1090,8 @@ run_backend_integration_tests() {
 
         if [ -f $WORKSPACE/integration/docker-compose.enterprise.yml ]; then
             cd $WORKSPACE/integration/backend-tests && \
-                PYTEST_ARGS="-k 'not Multitenant'" ./run && \
-                PYTEST_ARGS="-k Multitenant" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml || \
+                PYTEST_ARGS="-k 'not Enterprise and not Multitenant'" ./run && \
+                PYTEST_ARGS="-k Enterprise" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml || \
```

Comment:
I think we need here an "or Multitenant", as this script will still run on old integration versions (for instance next 2.1.0 release).

Reply:
I see now your commit message... Make sense :)

Reply:
ehh, I was hoping I would able to slide over this one ;) unfortunately, we can't do that, and it's tricky:

the old -Multitenant tests don't know anything about useradm-enterprise binary; they can only use 'useradm'. so integration versions that mix those two (current - for some time) are untestable in this way. only ST tests can be ran for them.

Reply:
so in other words, the logic goes as this:
- if we have the enterprise layer - try running -Enterprise tests, because nothing else has hope of running; even if we have -Multitenant tests, just skip them
- single tenant can be ran by ignoring both Enterprise and Multitenant
- if there's no enterprise layer yet: we could technically run -Multitenant tests (because we're sure they are there - introduced before the layer), but we don't really want to; we're running tests for an old release, and this means we're fixing something for existing onprem users with an older release, so save effort

Reply:
Yes. I understood that from your commit message. :+1: